app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from typing import List
+
+
+def calculate_team_total_rating(team: List[Elf | Dwarf]) -> int:
```

Comment:
The type hint for the `team` parameter should be `List[Player]` instead of `List[Elf | Dwarf]` to match the task description, which specifies that the function should take a list of `Player`s.